lib/rubocop/cop/team.rb

```diff
@@ -31,6 +31,14 @@ def initialize(cops, config = nil, options = nil)
         validate_config
       end
 
+      # @return [Team]
+      def self.new(cop_or_classes, config, options = nil)
+        # Support v0 api:
+        return mobilize(cop_or_classes, config, options) if cop_or_classes.first.is_a?(Class)
```

Comment:
@marcandre thanks for ripping this out so quickly. However after upgrading to `0.85.0` and trying your changes, I locally I'm getting:

```
NoMethodError: undefined method `first' for <RuboCop::Cop::Registry:0x00007f939c949dc8>
```

This is with `erb_lint` which started out here https://github.com/rubocop-hq/rubocop/issues/8077

```
---
linters:
  ErbSafety:
    enabled: false
  Rubocop:
    enabled: true
    rubocop_config:
      inherit_from:
        - .rubocop.yml
```

If I remove `Rubocop` from the above it runs fine.

Here's what I get from the arg `cop_or_classes` https://gist.github.com/darrenterhune/aa0b9b3a4886d4708b6c41e76cad4d4f

Reply:
Indeed, my patch relies on my previous patch #8079: `include Enumerable` will define `first` for `Registry`.